lib/impure/db_postgres.nim

```diff
@@ -234,6 +234,10 @@ proc len*(row: InstantRow): int32 {.inline.} =
   ## returns number of columns in the row
   pqNfields(row.res)
 
+proc getColumnName*(row: InstantRow, col: int32) : string {.inline.} =
```

Comment:
In this case, the signature of 
proc pqfname*(res: PPGresult, field_num: int32)
in postgres.nim must change to
proc pqfname*(res: PPGresult, field_num: int)